sdk/securityinsights/Azure.ResourceManager.SecurityInsights/tests/TestCase/ActionResponseCollectionTests.cs

```diff
@@ -0,0 +1,105 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Threading.Tasks;
+using Azure.Core;
+using Azure.Core.TestFramework;
+using Azure.ResourceManager.Resources;
+using Azure.ResourceManager.Resources.Models;
+using Azure.ResourceManager.SecurityInsights.Models;
+using Azure.ResourceManager.OperationalInsights.Models;
+using Azure.ResourceManager.SecurityInsights.Tests.Helpers;
+using NUnit.Framework;
+using Azure.ResourceManager.OperationalInsights;
+
+namespace Azure.ResourceManager.SecurityInsights.Tests.TestCase
+{
+    public class ActionResponseCollectionTests : SecurityInsightsManagementTestBase
+    {
+        public ActionResponseCollectionTests(bool isAsync)
+            : base(isAsync)//, RecordedTestMode.Record)
+        {
+        }
+        private async Task<ResourceGroupResource> GetResourceGroupAsync()
+        {
+            var resourceGroup = await CreateResourceGroupAsync();
+            return resourceGroup;
+        }
+        #region Workspace
+        private OperationalInsightsWorkspaceCollection GetWorkspaceCollectionAsync(ResourceGroupResource resourceGroup)
+        {
+            return resourceGroup.GetOperationalInsightsWorkspaces();
+        }
+        private async Task<OperationalInsightsWorkspaceResource> GetWorkspaceResourceAsync(ResourceGroupResource resourceGroup)
+        {
+            var workspaceCollection = GetWorkspaceCollectionAsync(resourceGroup);
+            var workspaceName1 = groupName + "-ws";
+            var workspaceInput = GetWorkspaceData();
+            var lrow = await workspaceCollection.CreateOrUpdateAsync(WaitUntil.Completed, workspaceName1, workspaceInput);
+            OperationalInsightsWorkspaceResource workspace = lrow.Value;
+            return workspace;
+        }
```

Comment:
Should we exclude the dependency creation from the recordings? @Yao725 could you please comment on this?

Reply:
fixed